nixos/modules/services/web-apps/jitsi-meet.nix

```diff
@@ -0,0 +1,497 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.jitsi-meet;
+  dataDir = "/var/lib/jitsi-meet";
+  attrsToArgs = a: concatStringsSep " " (mapAttrsToList (k: v: "${k}=${toString v}") a);
```

Comment:
`generators.toINI`?

Reply:
`toINI` has hardcoded newlines and sections, this one concatenates with spaces.